e2etests/stable_test.go

```diff
@@ -12,6 +12,35 @@ var testMarkdown string
 
 func testStable(t *testing.T) {
 	tcs := []testCase{
+		{
+			name: "class_and_sqlTable_border_radius",
+			script: `
+				a: {
+					shape: sql_table
+					id: int {constraint: primary_key}
+					disk: int {constraint: foreign_key}
+
+					json: jsonb  {constraint: unique}
+					last_updated: timestamp with time zone
+					
+					style: {
+						fill: red
+						border-radius: 10
+					}
+				}
+
+				b: {
+					shape: class
+
+					field: "[]string"
+					method(a uint64): (x, y int)
+					
+					style: {
+						border-radius: 10
+					}
+				}
```

Comment:
can you also add 
```
c: { shape: class }
d: { shape: sql_table }
```

people often use these without bodies, e.g. https://github.com/Greenroom-Robotics/ros-d2#example-usage

I thinkk the behavior should be the same as you have it now, which is the top corners are rounded but the bottom are not. but i'm curious how it looks

Reply:
@alixander fixed the logic for class and table without bodies with testcases. still trying to fix the env issue i got, do you know why in our ci process we need to our `/usr/bin/gem` ?

Reply:
The CI process doesn't. This is the leading theory: https://github.com/terrastruct/d2/issues/991#issuecomment-1458415718

Reply:
okay found the issue. fixed here: https://github.com/terrastruct/d2/pull/1002 . once you rebase with master, error should go away.